torch_geometric/nn/conv/gps_conv.py

```diff
@@ -55,32 +54,49 @@ class GPSConv(torch.nn.Module):
         norm_kwargs (Dict[str, Any], optional): Arguments passed to the
             respective normalization function defined by :obj:`norm`.
             (default: :obj:`None`)
+        attn_type (str): Global attention type, :obj:`softmax` or
+            :obj:`performer`. (default: :obj:`softmax`)
+        attn_kwargs (Dict[str, Any], optional): Arguments passed to the
+            attention layer. (default: :obj:`None`)
     """
     def __init__(
         self,
         channels: int,
         conv: Optional[MessagePassing],
         heads: int = 1,
         dropout: float = 0.0,
-        attn_dropout: float = 0.0,
```

Comment:
Why is this argument dropped? I think we should keep this and forward the argument to both `MultiheadAttention` and `PerformerAttention`.

Reply:
IMO this dropout is specific to different attention mechanism, so it might be better to put this into the `attn_kwargs`. Also, it's possible that some other attention mechanism might not need `attn_dropout`. But drop this may break some current script if user is using the `GPSConv` with `attn_dropout`. WDYT?